APIP_Micro/main.py

```diff
@@ -0,0 +1,2 @@
+def app(event):
+    return "Hello, world!"
```

Comment:
That is generated by deta and I don't we should edit those files ;) I don't really know if those should be there either but we will migrate to docker and some cloud provider later so no worries it will be deleted in the future]